dev-libs/spdlog/spdlog-1.4.2.ebuild

```diff
@@ -10,29 +10,31 @@ HOMEPAGE="https://github.com/gabime/spdlog"
 
 if [[ ${PV} == *9999 ]]; then
 	inherit git-r3
-	EGIT_REPO_URI="https://github.com/gabime/spdlog"
+	EGIT_REPO_URI="https://github.com/gabime/${PN}"
 else
-	SRC_URI="https://github.com/gabime/spdlog/archive/v${PV}.tar.gz -> ${P}.tar.gz"
-	KEYWORDS="~amd64"
+	SRC_URI="https://github.com/gabime/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="amd64"
 fi
 
 LICENSE="MIT"
 SLOT="0/1"
-IUSE="test"
+IUSE="static-libs test"
+RESTRICT="!test? ( test )"
 
+BDEPEND="
+	virtual/pkgconfig
+"
 DEPEND="
-	>=dev-libs/libfmt-5.0.0
+	>=dev-libs/libfmt-5.3.0
 "
 RDEPEND="${DEPEND}"
 
-PATCHES=( "${FILESDIR}/${PN}-1.0.0-unbundle-fmt.patch" )
-
 src_configure() {
-	rm -r include/spdlog/fmt/bundled || die
```

Comment:
shouldn't we removed the bundled fmtlib btw?

Reply:
I don't think so, they don't end up in the image if you force the use of an external fmt, which we do.

Reply:
well, at least this way you make sure it does not build against bogus headers by accident

Reply:
nice. The last day I saw bundled files being installed on the image, but definitely was my mistake :P

Reply:
I'll reintroduce in `src_prepare` and see how it goes

Reply:
Worked well, fixed :)